app/models/page_view.rb

```diff
@@ -1,6 +1,3 @@
-# @note When we destroy the related article, it's using dependent:
```

Comment:
Why did this comment get removed?

Reply:
Thank you, looks like mistreated merging.